dbms/src/Server/Server.cpp

```diff
@@ -1095,11 +1091,16 @@ int Server::main(const std::vector<std::string> & /*args*/)
     auto main_config_reloader = std::make_unique<ConfigReloader>(
         config_path,
         [&](ConfigurationPtr config) {
+            LOG_INFO(log, "run main config reloader");
```

Comment:
main_config_reloader will check whether `config.toml` is changed periodically, if `config.toml` is modified, it will call  the lambda function set below.

Reply:
it should be debug level